src/sentry/apidocs/examples/organization_examples.py

```diff
@@ -37,11 +37,14 @@ class OrganizationExamples:
                     "hasSessions": True,
                     "hasProfiles": True,
                     "hasReplays": True,
+                    "hasMinifiedStackTrace": False,
```

Comment:
This had to be added to this OpenApiExample to comply with adding the field to the Project Model typed dict